docs/testbed/README.testbed.Example.Config.md

```diff
@@ -0,0 +1,143 @@
+# Example of Testbed Configuration
+Here is an example of how to manually get the testbed information when you want to allocate some tests on the DUTs. It includes how to get the testbed IP address, location, and how to find where is the PTF docker from the configuration files.
+
+## Testbed Inventory
+
+- [```ansible/lab```](/ansible/lab): Include all lab DUTs, fanout switches, and testbed server topologies
+- [```ansible/veos```](/ansible/veos): all servers and VMs
+- [```ansible/testbed.csv```](/ansible/testbed.csv) is the topology configuration file for the testbed (Will be deprecated).
+- [```ansible/testbed.yaml```](/ansible/testbed.yaml) is the topology configuration file for the testbed .
+- [```ansible/str```](/ansible/str) [```ansible/str2```](/ansible/str2)  is the environment configuration files for the testbed.
```

Comment:
`str` and `str2` are internal inventory files. Suggest to change it to `ansible/inventory` and `ansible/lab` which are example inventory files in public repo. These two files are just two examples. You can update either of them to match the actual devices in your lab or create new inventory file for your lab.